inst/Upgrades.qml

```diff
@@ -0,0 +1,100 @@
+import QtQuick		2.12
+import JASP.Module	1.0
+
+Upgrades
+{
+	Upgrade
+	{
+		functionName: 	"MixedModelsLMM"
+		fromVersion:	"0.14.3"
+		toVersion:		"0.15"
+
+		ChangeRename { from: "bootstrap_samples";	to: "bootstrapSamples"; }
+		ChangeSetValue
+		{
+			name:		"plotsTheme"
+			condition:	function(options) { return options["plotsTheme"] == "jtools::theme_apa"; }
+			jsonValue:	"theme_apa"
+		}
+		ChangeSetValue
+		{
+			name:		"plotsTheme"
+			condition:	function(options) { return options["plotsTheme"] == "ggpubr::theme_pubr"; }
+			jsonValue:	"theme_pubr"
+		}
```

Comment:
You could reduce the code duplication by changing the option values in a single function:
```suggestion
		ChangeJS
		{
			name:		"plotsTheme"
			jsFunction:	function(options) 
			{
				switch(options["plotsTheme"])
				{
					case "jtools::theme_apa":	return "theme_apa";
					case "ggpubr::theme_pubr":	return "theme_pubr";
					default:			return options["plotsTheme"]
				}
			}
		}
```